examples/src/main/scala/Protocol.scala

```diff
@@ -30,7 +30,8 @@ import codecs._
 @GenClient(remotely.Protocol.empty.codec[Int].specify1[Int,Int]("fac").signatures)
   object FacClient
 
-@GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
+// TODO(ahjohannessen): @stew <- Compiling Foo fails somehow?
+// @GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
```

Comment:
This thing fails if included in code for some reason, same problem as reported 27 days ago in the initial pull request [https://github.com/oncue/remotely/pull/34]